pages/myProfile/index.js

```diff
@@ -14,46 +16,110 @@ const myProfile = ({loginName, userArray}) => {
     },
   };
   // parses loginName String to Object to be able to compare them in the checkLoggedInUser Method
-  const loginNameObject = {name: `${loginName}`};
-  const checkLoggedInUser = userArray.filter(user => {
-    return user.name == loginNameObject.name;
+  const [ownProfileUser, setOwnProfileUser] = useState();
+  useEffect(() => {
+    //first
+    const loginNameObject = {name: `${loginName}`};
+    const checkLoggedInUser = userArray.filter(function (user) {
```

Comment:
If you want to find a single user, better use the find method:

```suggestion
    const checkLoggedInUser = userArray.find(function (user) {
```

Reply:
Fixed and pushed new version.